ktor-server/ktor-server-tests/common/test/io/ktor/tests/server/plugins/ConditionalHeadersTests.kt

```diff
@@ -157,4 +157,11 @@ class ETagsTest {
         }
         assertEquals(HttpStatusCode.PreconditionFailed, result.status)
     }
+
+    @Test
+    fun testInvalidWeakStarETag() = withConditionalApplication {
+        assertFailsWith<IllegalArgumentException> {
+            EntityTagVersion(EntityTagVersion.STAR.etag, weak = true)
+        }
+    }
```

Comment:
Minor. Probably `withConditionalApplication` can be removed
```suggestion
    fun testInvalidWeakStarETag() {
        assertFailsWith<IllegalArgumentException> {
            EntityTagVersion(EntityTagVersion.STAR.etag, weak = true)
        }
    }
```